arviz/stats/stats.py

```diff
@@ -1269,6 +1276,11 @@ def summary(
     extra_metrics = []
     extra_metric_names = []
 
+    if stat_focus is not None:
+        if stat_focus == "median":
+            stat_funcs = {"median": np.median, "mad": st.median_abs_deviation}
+            extend = False
```

Comment:
Does summary only return the median and the mad when `stat_focus="median"`?

Reply:
Yes currently it gives median and mad only, I have added the example in the Description.